Change whitespace marks

viccuad opened the pull request:
This PR changes the whitespace-marks foregrounds (tab, spaces, newlines) to one
that is between the default background color and the comments's color.

That way one can discern them easily, yet you don't confuse them with
comments.

eg: 

![whitespace-emacs](https://cloud.githubusercontent.com/assets/2196685/12992473/f84bb58e-d114-11e5-8bec-ae0ca2c41307.png)

bkruczyk commented:
I'm okay with this.
Please amend your pull request so that it only contains appropriate commit (`a19977e`).

viccuad committed:
Change whitespace-marks foregrounds to less visible

Change the whitespace-marks foregrounds (tab, spaces, newlines) to one
that is between the default background color and the comments's color.

That way one can discern them easily, yet you don't confuse them with
comments.

viccuad commented:
I have fixed the branch, it should look good now :).

Cheers and thanks!

bkruczyk commented:
Thank you for contributing :)
